src/main/java/io/appform/dropwizard/discovery/bundle/resolvers/NodeInfoResolver.java

```diff
@@ -0,0 +1,10 @@
+package io.appform.dropwizard.discovery.bundle.resolvers;
+
+import io.appform.dropwizard.discovery.bundle.ServiceDiscoveryConfiguration;
+import io.appform.ranger.common.server.ShardInfo;
+
+public interface NodeInfoResolver {
```

Comment:
Could we add some Javadocs here to indicate the purpose of this class?

Reply:
I thought the method was self explanatory. Should we add that to readme of the repo instead, so folks using it know how to init it and why to?